src/main/resources/admin/tools/content-type-manager/components/structure/component-content-type.html

```diff
@@ -1,9 +1,11 @@
 <div th:fragment="component-content-type" th:remove="tag">
   <template>
-    <v-card>
+    <v-card v-if="fileName">
       <v-row>
         <v-col cols="10">
-          <v-card-title>{{ inputType }}</v-card-title>
+          <v-card-title>
+            <v-chip color="green">{{ fileName }}</v-chip>
```

Comment:
![image](https://user-images.githubusercontent.com/3340349/143256532-a8a61e6b-b743-486e-a965-6e347b787638.png)
The test is a little smal. Could you add a <h1> element or simple just replace the old headline?

Reply:
Sorry, but I'm not sure if I understand what do you mean here. Can you please try to explain it differently?

Reply:
I guess he meant "text is a little small" :)

I don't think we need to style it with green background, it looks like a button now. Can you change it back to how it was before? Just a normal header, without excessive styling.

Reply:
Yeah markup styling my comment wierdly. Updated